python/docs/source/getting_started/installation.rst

```diff
@@ -1,114 +0,0 @@
-..  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Ah .. looks like there are too many diff and it doesn't know that it was renamed ..